build.gradle.kts

```diff
@@ -44,23 +44,19 @@ configure<JavaPluginExtension> {
 }
 
 gradlePlugin {
+    website.set(ProjectInfo.URL)
+    vcsUrl.set(ProjectInfo.URL)
     plugins {
         create("buf") {
             id = "build.buf"
             implementationClass = "build.buf.gradle.BufPlugin"
             displayName = ProjectInfo.NAME
             description = ProjectInfo.DESCRIPTION
+            tags.set(listOf("protobuf", "kotlin", "buf"))
         }
     }
 }
 
-pluginBundle {
```

Comment:
This no longer works in 8.x - updated based on example here: https://plugins.gradle.org/docs/publish-plugin#examples